### ISSUE TITLE
Genes present in ontology but not in the models

### ISSUE THREAD

**Issue by CodeNoobje** [arpcard/amr_curation#12]
**General**
We have a dataset of annotated resistance genes and we ran RGI on them. Interestingly, we noticed that RGI misidentified some genes even though these were mentioned in the CARD ontology.

**Example**
To inspect the cause we  in detail looked at a single sequence 100% identical to OXA-402 in the [Beta lactambase database](http://www.bldb.eu/). RGI falsely identified our sequence as OXA-80. We found that [OXA-402 was present in the ontology](https://card.mcmaster.ca/ontology/37988) but it is not in the protein homolog model (Downloaded the latest FASTA from CARD). Therefore RGI can never find the prefect match (100% identity) with OXA-402 and picks the OXA-80. We are wondering why for example OXA-402 is not present in the protein homolog model. This is not the only gene though, we also noticed it for OXA-140 and  it probably applies to more genes.

**Comment by arpcard** [comment]
Hello @CodeNoobje, thanks for highlighting this. In the past, CARD required a peer-reviewed publication in PubMed to be associated with any sequence we curated. Unfortunately, a large number of beta-lactamases did not have corresponding publications or these were not cited when we worked from the Bush & Jakoby online list to boot-strap entry of beta-lactamases in CARD. NCBI is now curating this and we are also working through the backlog to add these sequences. We got several hundred done in last month's release, but hand't got to the OXAs yet. These will be added soon.
